src/main/java/org/openstreetmap/atlas/geography/atlas/items/Relation.java

```diff
@@ -308,7 +307,7 @@ public LocationIterableProperties toGeoJsonBuildingBlock()
         final Optional<String> shardName = getAtlas().metaData().getShardName();
         shardName.ifPresent(shard -> tags.put("shard", shard));
 
-        return new GeoJsonBuilder.LocationIterableProperties(Location.CENTER, tags);
+        return new GeoJsonBuilder.LocationIterableProperties(bounds().center(), tags);
```

Comment:
This is a great improvement! I was just thinking that it might be even better if it matched how `Relation.asGeoJsonGeometry()` handles relation geometry.

Reply:
I agree. The only issue is that ```LocationIterableProperties``` takes a ```Iterable<Location>``` which cannot represent our MultiPolygon object, which the other Geojson representation makes if possible. This is part of the reason  ```LocationIterableProperties``` is deprecated and we're working it out, but while we are it should work as intended.